src/tiered/tiered_handle.c

```diff
@@ -698,9 +725,11 @@ __tiered_open(WT_SESSION_IMPL *session, const char *cfg[])
     WT_ERR_NOTFOUND_OK(ret, true);
 
     /* Open tiers if we have them, otherwise initialize. */
-    if (tiered->current_id != WT_TIERED_OBJECTID_NONE)
+    if (tiered->current_id != WT_TIERED_OBJECTID_NONE) {
         WT_ERR(__tiered_init_tiers(session, tiered, &tierconf));
-    else {
+        if (session->import_list != NULL)
+            WT_ERR(__wt_tiered_switch(session, config));
```

Comment:
This code, although small probably deserves a comment.

Reply:
Done.